ember/app/ui/components/identity-emails/component.js

```diff
@@ -65,8 +66,14 @@ export default class IdentityEmailsComponent extends Component {
       this.onUpdate();
       this.changeset = null;
     } catch (error) {
+      const unique = error.errors?.find(({ code }) => code === "unique");
+      if (unique) {
+        changeset.addError("email", unique.detail);
```

Comment:
why are you using `changeset.addError` here and later `addError(changeset)`?

Reply:
Is the `addError` function for adding exceptions to the changeset? If yes then maybe renaming it to `addException` or `addExceptionError` would make this a bit more understandable.

Reply:
I don't like calling the errors exceptions. Javascript doesn't have exceptions. And if I changed this function, would I also rename the `formatError` utils function?

Reply:
Fair but do you see my point of this being a bit confusing?

Reply:
I do. I would have said. Shit happens :smile: But I'll look if I find a better wording.

Reply:
I added a comment to the manual error here and changed the name of the utility function to `applyError`. Loosely based on [function's apply](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/apply) as the function applies `changeset.addError` for a list of errors.